podman/client.py

```diff
@@ -62,15 +62,21 @@ def __init__(self, **kwargs) -> None:
 
         api_kwargs = kwargs.copy()
 
-        if "connection" in api_kwargs:
-            connection = config.services[api_kwargs.get("connection")]
-            api_kwargs["base_url"] = connection.url.geturl()
-
-            # Override configured identity, if provided in arguments
-            api_kwargs["identity"] = kwargs.get("identity", str(connection.identity))
-        elif "base_url" not in api_kwargs:
-            path = str(Path(get_runtime_dir()) / "podman" / "podman.sock")
-            api_kwargs["base_url"] = "http+unix://" + path
+        if not api_kwargs:
+            if "Connection" in config.attrs:
+                default_connection_name = config.attrs["Connection"]["Default"]
+                connection = config.attrs["Connection"]["Connections"][default_connection_name]
+                api_kwargs["base_url"] = connection["URI"]
+                api_kwargs["identity"] = connection["Identity"]
+        else:
+            if "connection" in api_kwargs:
+                connection = config.services[api_kwargs.get("connection")]
+                api_kwargs["base_url"] = connection.url.geturl()
+                # Override configured identity, if provided in arguments
+                api_kwargs["identity"] = kwargs.get("identity", str(connection.identity))
+            elif "base_url" not in api_kwargs:
```

Comment:
@jwhonce any reason why this is technically the default case (this will always be true if api_kwargs is empty) and not the contents we get from the podman config?